### PR TITLE
[REG] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

File: base/applications/cmdutils/reg/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,46 +1,50 @@
 /*
- * Translators: Claudia Cotună
- *              Michael Stefaniuc
- *              Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS Registry Command Line Utils
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011 Claudia Cotună
+ *              Copyright 2011-2015 Michael Stefaniuc
+ *              Copyright 2011-2016 Ștefan Fulea <stefan.fulea@mail.com>
```
Comment (binarymaster) [inline base/applications/cmdutils/reg/lang/ro-RO.rc:7]:
```suggestion
 * PROJECT:     ReactOS Registry Console Tool
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Romanian resource file
 * TRANSLATORS: Copyright 2010 Claudia Cotună <special4ti@gmail.com>
 *              Copyright 2010 Michael Stefaniuc <mstefani@redhat.com>
 *              Copyright 2014-2018 Ștefan Fulea <stefan.fulea@mail.com>
```

**Comment by GeoB99** [comment]
@AndreiMiloiu Did you post this PR as a draft by mistake? It doesn't look like there's any WIP left to address here.

**Comment by AndreiMiloiu** [comment]
No, I still have to translate here. I didn't have enough time to translate it at all. I'll finish it these days.

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: base/applications/cmdutils/reg/lang/ro-RO.rc [hunk 2]
```diff
@@ -104,84 +108,84 @@ For help on a specific operation, type:\n\
...
+     Copiază toate subcheile și valorile din <cheia1> în <cheia2>.\n\n\
   /f\n\
-     Overwrite all registry data in <key2> without prompting for confirmation.\n\
-     This option does not modify subkeys and values that only exist in <key2>.\n\n"
+     Suprascrie toate datele registrului în <cheia2> fără să se necesite confirmare.\n\
+     Această opțiune nu modifică subcheile și valorile ce există doar în <cheia2>.\n\n"
 
-    STRING_COPY_SRC_DEST_SAME, "reg: The source and destination keys cannot be the same\n"
-    STRING_COPY_CONFIRM, "The value '%1\\%2' already exists in the destination key. Do you want to overwrite it?"
+    STRING_COPY_SRC_DEST_SAME, "reg:Cheile sursă și destinație nu pot fi aceleași\n"
```
Comment (JoachimHenze) [inline base/applications/cmdutils/reg/lang/ro-RO.rc:189]:
```suggestion
    STRING_COPY_SRC_DEST_SAME, "reg: Cheile sursă și destinație nu pot fi aceleași\n"
```
To match other STRING_* which do start with the "reg: "-pattern.

**Pull request merged**